account_financial_report_qweb/report/general_ledger.py

```diff
@@ -330,7 +325,7 @@ def _get_account_sub_subquery_sum_amounts(
             """
         sub_subquery_sum_amounts += """
         LEFT JOIN
-            res_currency c ON ml.currency_id = c.id
+            res_currency c ON ml.currency_id = c.id        
```

Comment:
Why did you add white spaces? 
I had to remove them for https://github.com/OCA/account-financial-reporting/pull/390 to be accepted by Travis's LINT, so I think they shouldn't be there.

Reply:
Fixed!